homestar-runtime/src/network/swarm.rs

```diff
@@ -57,6 +57,18 @@ pub(crate) async fn new(settings: &settings::Node) -> Result<Swarm<ComposedBehav
     // Listen-on given address
     swarm.listen_on(settings.network.listen_address.to_string().parse()?)?;
 
+    // Dial bootstrap nodes specified in settings. Failure here shouldn't halt node startup.
+    for bootstrap_addr in &settings.network.bootstrap_addresses {
```

Comment:
minor, but let's maybe hide this into an impl function and call it `fn bootstrap`? This may include the subscribe and listen steps.

Reply:
renaming to an even more generic `swarm_init`. eventally I'd like to put all startup things in it but we need to define an interface to work with networking.